reference/features/search_parameters.md

```diff
@@ -1,133 +1,162 @@
+    },
+  ],
+  "query":"shifu"
+}
+```
+
 ### Placeholder search
 
-When `q` isn't specified, a **placeholder search** is performed. Placeholder search allows you to find documents without providing any search terms.
+When `q` isn't specified, MeiliSearch performs a **placeholder search**. A placeholder search returns all documents in an index, sorted by that index's [custom ranking rules](/reference/features/settings#custom-ranking-rule).
 
-Placeholder search returns the documents that best match other search parameters, sorted according to that index's [ranking rules](/reference/features/settings.md#ranking-rules). This feature also supports [faceting](/reference/features/faceted_search.md) and [filtering](/reference/features/filtering.md)
+If the index has no custom ranking rules, the results are returned in the order of their internal database position.
 
-## Offset
+::: tip
+Placeholder searches are particularly effective when combined with [faceting](/reference/features/faceted_search) and [filtering](/reference/features/filtering.md).
+:::
 
-A number of **documents to skip**.
+## Offset
 
-`offset=<Integer>`
+**Parameter**: `offset`
+**Expected value**: any positive integer
+**Default value**: `0`
 
-- `<Integer>` (Optional, positive integer, defaults to `0`)
+Sets the starting point in the search results, effectively skipping over a given number of documents.
 
-  If the value of the parameter `offset` is _n_, _n_ first documents to skip. This is helpful for **pagination**.
+::: tip
+This parameter is often used together with `limit` in order to paginate results.
+:::
 
-#### Example
+### Example
 
-If you want to skip the **first** document, set `offset` to `1`.
+If you want to skip the **first** result in a query, set `offset` to `1`:
 
 <CodeSamples id="search_parameter_guide_offset_1" />
 
 ## Limit
 
-Set a **limit to the number of documents returned** by search queries.
-
-`limit=<Integer>`
+**Parameter**: `limit`
+**Expected value**: any positive integer
+**Default value**: `20`
 
-- `<Integer>` (Optional, positive integer, defaults to `20`)
+Sets the maximum number of items returned by a single query.
 
-  If the value of the parameter `limit` is _n_, there will be _n_ documents in the search query response. This is helpful for **pagination**.
+::: tip
+This parameter is often used together with `offset` in order to paginate results.
+:::
 
-#### Example
+### Example
 
-If you want to get only **two** documents, set `limit` to `2`.
+If you want your query to return only **two** documents, set `limit` to `2`:
 
 <CodeSamples id="search_parameter_guide_limit_1" />
 
 ## Filters
 
-`filters=<String>`
+**Parameter**: `filters`
+**Expected value**: a string containing a query expression
+**Default value**: `null`
 
-Specify a filter to be used with the query. See our [dedicated guide](/reference/features/filtering.md).
+Filters query results using [MeiliSearch's query language](/reference/features/filtering.md#the-query-language).
```

Comment:
Had to do a double take on this one as I briefly read query as a verb and filters as a noun without the context.

Reply:
Oh wow, I didn't realize it could be read that way but you're absolutely right. Maybe this phrasing would be better.

`Applies a filter to the search process. The supplied string must use MeiliSearch's [filter syntaxes](/reference/features/filtering.md#the-query-language)`